src/app/explorer/info.tsx

```diff
@@ -3,24 +3,46 @@ import { useParams } from "react-router-dom";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import { ResourceInstance } from "@/state/fetch";
 import { ResourceSchema } from "@/state/openapi";
+import { useAppSelector } from "@/hooks/store";
+import { selectChildResources } from "@/state/store";
+import ResourceListPage from "./resource_list";
+
+type ResourceProperties = {
+    path: string;
+    [key: string]: string | number | boolean;
+}
 
 type InfoPageProps = {
     resource: ResourceSchema
 }
 
 export default function InfoPage(props: InfoPageProps) {
-    const { resourceId }= useParams();
-    const [state, setState] = useState<ResourceInstance>({});
+    const params = useParams();
+    const [state, setState] = useState<ResourceInstance | null>(null);
+    
+    const childResources = useAppSelector((globalState) => 
+        state ? selectChildResources(globalState, props.resource, (state.properties as ResourceProperties)?.path?.split('/').pop() || '') : []
```

Comment:
nit: factoring out the`state.properties as ....` to a variable would be a bit easier to read.